tensorflow/python/kernel_tests/nn_ops/atrous_conv2d_test.py

```diff
@@ -196,6 +208,18 @@ def testAtrousConv2DTransposeForward(self):
                     x, f_up, y_shape, strides=[1, 1, 1, 1], padding=padding)
                 self.assertAllClose(y1, y2, rtol=1e-3, atol=1e-3)
 
+  @test_util.run_deprecated_v1
+  def testAtrousConv2DTransposeInvalid(self):
+    with self.session():
```

Comment:
Does this needed the `run_deprecated_v1` and `with self.session()`?  We're trying to slowly update all these tests to run in V2 mode.